content/en/docs/spinnaker/terraform-use-integration.md

```diff
@@ -30,14 +30,15 @@ To use the stage, perform the following steps:
 5. Configure the Terraform Integration stage.
     The available fields may vary slightly depending on what you configure for the stage:
     * **Basic Settings**
-      * **Terraform Version**:  Terraform version to use. All Terraform stages within a pipeline that modify state (apply, output, destroy) must use the same version.
+      * **Terraform Version**:  Terraform version to use. All Terraform stages within a pipeline that modify state (apply, output, destroy) must use the same version. If you use a remote backend, the minimum supported version is 0.12.0 and you must select the same Terraform version that your Terraform Cloud/Enterprise runs.
```

Comment:
Might be clearer to say `if you use _the_ remote backend...`

Reply:
It might also be clearer to say "...and you must select the same Terraform version your Terraform Cloud/Enterprise workspace is configured to use."